nvflare/app_opt/xgboost/histogram_based_v2/sec/client_handler.py

```diff
@@ -54,6 +69,11 @@ def __init__(self, key_length=1024, num_workers=10):
         self.feature_masks = None
         self.aggregator = Aggregator()
         self.aggr_result = None  # for label client: computed aggr result based on clear-text clear_ghs
+        self.tenseal_context_file = tenseal_context_file
+        self.tenseal_context = None
+
+        if tenseal_imported:
+            decomposers.register()
```

Comment:
Should we do this in this component?
Is there a better place to auto register decomposers in app_opt?

Reply:
Will take care of this once Yuhong's change is done.